docs/3.api/4.advanced/1.hooks.md

```diff
@@ -60,6 +60,7 @@ Hook                     | Arguments                  | Description
 `nitro:config`           | `nitroConfig`              | Called before initializing Nitro, allowing customization of Nitro's configuration.
 `nitro:init`             | `nitro`                    | Called after Nitro is initialized, which allows registering Nitro hooks and interacting directly with Nitro.
 `nitro:build:before`     | `nitro`                    | Called before building the Nitro instance.
+`nitro:build:assets`     | `nitro`                    | Called after copying public assets into `nitro.options.output.publicDir`. Allows modifying public assets before Nitro server is built.
```

Comment:
On small note, i wouln't recommand `publicDir` in docs as it might break presets depending on structure.